hca/staging/cli/select_command.py

```diff
@@ -0,0 +1,35 @@
+import re
+
+from .config_store import ConfigStore
+from .staging_area_urn import StagingAreaURN
+
+
+class SelectCommand:
+
+    def __init__(self, args):
+        self.config = ConfigStore()
+        if re.search(':', args.urn_or_alias):  # URN
+            self._save_and_select_area_by_urn(args.urn_or_alias)
+        else:  # alias
+            self._select_area_by_alias(args.urn_or_alias)
+        self.config.save()
+
+    def _save_and_select_area_by_urn(self, urn_string):
+        urn = StagingAreaURN(urn_string)
+        self.config.add_area(urn)
+        self._select_area(urn.uuid)
+
+    def _select_area_by_alias(self, alias):
+        matching_areas = [uuid for uuid in self.config.areas() if re.match(alias, uuid)]
+        if len(matching_areas) == 0:
+            print("Sorry I don't recognize area \"%s\"" % (alias,))
+        elif len(matching_areas) == 1:
+            self._select_area(matching_areas[0])
+        else:
+            print("\"%s\" matches more than one area, please provide more characters." % (alias,))
+
+    def _select_area(self, area_uuid):
+        self.config.select_area(area_uuid)
+        print("Staging area %s selected." % area_uuid)
+        print("You may refer to this staging area in future using the first few characters, e.g. \"%s\"" %
+              area_uuid[0:3])
```

Comment:
strictly speaking, this is not true.  you'd need to run it through `_select_area_by_alias` to see if it's a unique prefix.